### PR TITLE
Fix W3C errors

### PR DESCRIPTION
**Pull request by l33d4n**
## Homepage: 

Full list: https://validator.w3.org/nu/?doc=https%3A%2F%2Fgrin.mw%2F

image

## Get Started:

Full list: https://validator.w3.org/nu/?doc=https%3A%2F%2Fgrin.mw%2Fget-started

image

## Download:

Full list: https://validator.w3.org/nu/?doc=https%3A%2F%2Fgrin.mw%2Fdownload

image

## Governance:

Full list: https://validator.w3.org/nu/?doc=https%3A%2F%2Fgrin.mw%2Fgovernance

image

## Funding:

Full list: https://validator.w3.org/nu/?doc=https%3A%2F%2Fgrin.mw%2Fdownload

image

### CODE REVIEW

**Comment by quentinlesceller** [comment]
Hi @l33d4n thank you for the PR? Does this break anything?

**Comment by l33d4n** [comment]
No :) These small changes, they do not affect the structure of the website or its design.

Demo: https://l33d4n.github.io/site/

File: governance.md [hunk 1]
```diff
@@ -19,55 +19,55 @@ Project direction, sub-teams, security, general fund, cross-cutting concerns.
...
+        <a href="http://github.com/yeastplume" style="border-bottom: 0px ! important;"><img src="https://avatars.githubusercontent.com/yeastplume" alt="Michael Cordner" class="team-table-avatar"></a>
         <div class="team-table-details">
             <p class="team-table-title">Michael Cordner</p>
             <p class="team-table-text">github: <a href="http://github.com/yeastplume">yeastplume↗</a></p>
             <p class="team-table-text">keybase: <a href="https://keybase.io/yeastplume">@yeastplume↗</a></p>
         </div>
     </div>
     <div class="team-table-member">
-        <a href="http://github.com/quentinlesceller" style="border-bottom: 0px ! important;"><img src="https://avatars.githubusercontent.com/quentinlesceller" class="team-table-avatar"></a>
+        <a href="http://github.com/quentinlesceller" style="border-bottom: 0px ! important;"><img src="https://avatars.githubusercontent.com/quentinlesceller" alt="Quentin Le Scellerr" class="team-table-avatar"></a>
```
Comment (quentinlesceller) [inline governance.md:70]: Just noticed the typo here :) 
```suggestion
        <a href="http://github.com/quentinlesceller" style="border-bottom: 0px ! important;"><img src="https://avatars.githubusercontent.com/quentinlesceller" alt="Quentin Le Sceller" class="team-table-avatar"></a>
```

File: index.html [hunk 2]
```diff
@@ -35,7 +35,7 @@
       <span class="medium">Mimblewimble</span> leverages cryptography to allow past transaction data to be removed with no compromise on security. This avoids Grin collapsing under the weight of data having to be kept on chain.
     </p>
   </section>
-  <img class="section-2-cash" src="assets/images/cash.png" />
+  <img class="section-2-cash" alt="MimbleWimble" src="assets/images/cash.png" />
```
Comment (quentinlesceller) [inline index.html:38]:
```suggestion
  <img class="section-2-cash" alt="Mimblewimble" src="assets/images/cash.png" />
```

File: index.html [hunk 3]
```diff
@@ -22,7 +22,7 @@
       <span class="section-1-star">*</span>
     </div>
   </section>
-  <img class="section-1-lights" src="assets/images/img1@3x.png" />
+  <img class="section-1-lights" alt="Electronic transactions for all" src="assets/images/img1@3x.png" />
```
Comment (quentinlesceller) [inline index.html:25]: Unsure about this alt text maybe something more general. Isn't the primary purpose of `alt` text to describe the image to visitors who are unable to see them?

**Review by quentinlesceller — CHANGES_REQUESTED** [review]
Small comments that I missed from last time.

File: index.html (hunk 3 above)
Comment (l33d4n) [reply index.html:25]: You're right. Thanks!

Comment (quentinlesceller) [reply index.html:25]: something like `Protect yourself against electronic surveillance` ? Unsure trying to describe that imagine is hard.

Comment (quentinlesceller) [reply index.html:25]: Once this is fixed we can merge this

Comment (l33d4n) [inline index.html:25]:
```suggestion
  <img class="section-1-lights" alt="Protect yourself against electronic surveillance" src="assets/images/img1@3x.png" />
```

Comment (l33d4n) [reply index.html]: Done. Thanks!

**Review by quentinlesceller — APPROVED** [review]

**Comment by quentinlesceller** [comment]
Thank you for the PR @l33d4n 🚀.  Merging!

**Comment by l33d4n** [comment]
Great! Thanks for the review.

**Pull request merged**